src/BloomExe/Book/BookStorage.cs

```diff
@@ -656,9 +658,16 @@ private void UpdateSupportFiles()
 
 			//by default, this comes from the collection, but the book can select one, inlucing "null" to select the factory-supplied empty xmatter
 			var nameOfXMatterPack = _dom.GetMetaValue("xMatter", _collectionSettings.XMatterPackName);
-			var helper = new XMatterHelper(_dom, nameOfXMatterPack, _fileLocator);
-			UpdateIfNewer(Path.GetFileName(helper.PathToStyleSheetForPaperAndOrientation), helper.PathToStyleSheetForPaperAndOrientation);
 
+			try
+			{
+				var helper = new XMatterHelper(_dom, nameOfXMatterPack, _fileLocator);
+				UpdateIfNewer(Path.GetFileName(helper.PathToStyleSheetForPaperAndOrientation), helper.PathToStyleSheetForPaperAndOrientation);
+			}
+			catch (Exception error)
```

Comment:
John, I appreciate you point. But in this case, it seems too weak to warrant trying to come up with this list of OK errors versus Unexpected errors. Note,  the catch here doesn't swallow anything. Rather, it surfaces the error to the user in a way that works even when loading up dozens of books, any of which may have problems. The ErrorMessages property that is set here is used to do a couple things, including displaying the error contents where the book preview would normally be, and disabling editing of the book.